### PR TITLE
Moved fsharpqa/Conformance/BasicGrammarElements/Constants test cases to NUnit

### PR DESCRIPTION
**Pull request by ThorstenReichert**
Contributes to #7075.

### CODE REVIEW

File: tests/FSharp.Compiler.ComponentTests/ErrorMessages/InvalidNumericLiteralTests.fs [hunk 1]
```diff
@@ -5,16 +5,84 @@ namespace FSharp.Compiler.ErrorMessages.ComponentTests
...
+                FSharpErrorSeverity.Error, 599, (2, 11, 2, 12),"Missing qualification after '.'"
+            |]
+
+    [<Fact>]
+    let ``_52 is invalid numeric literal``() =
         CompilerAssert.TypeCheckSingleError
             """
-let foo = 1up // int
+let x = _52
             """
```
Comment (KevinRansom) [inline tests/FSharp.Compiler.ComponentTests/ErrorMessages/InvalidNumericLiteralTests.fs:51]: The test is checking that an identifier that starts with a numeric is detected as valid, and is a valid test case,  why the change?

Thanks

Kevin

Comment (ThorstenReichert) [reply tests/FSharp.Compiler.ComponentTests/ErrorMessages/InvalidNumericLiteralTests.fs:51]: The original test case for `1up` got merged as a `InlineData` into the `Theory` above, with other similar test cases from this migration.
The `_52` test case is from this migration (put it as separate, because it has a different behaviour, i.e. two error messages).

Comment (KevinRansom) [reply tests/FSharp.Compiler.ComponentTests/ErrorMessages/InvalidNumericLiteralTests.fs:51]: @ThorstenReichert 

sweet, thanks.
